andela/activo-web
# #158563733 User can set options for list fields

**What does this PR do?**
Adds the functionality for configuring options for a list field

**Description of Task to be completed?**
- create `addcategoryoptions` component to handle the creation of options
- create `displaycategoryoptions` component to handle the display of created options
- write tests for `addcategoryoptions` component
- write tests for `displaycategoryoptions` component
- install `uuid` to generate unique ids for items

**How should this be manually tested?**
- Fetch and checkout to remote branch
- Run `yarn` to install required dependencies
- Run `yarn start:dev` to start up the application.
- Go to the add category page.  On the additional attributes, select one of the available options from the drop-down.
- A text box should appear beneath. Enter the configuration options for that field, and press enter

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**
[#158563733](https://www.pivotaltracker.com/story/show/158563733)

**Screenshots (if appropriate)**
<img width="1280" alt="screen shot 2018-07-10 at 6 40 42 am" src="https://user-images.githubusercontent.com/24798364/42491300-39a8240c-840c-11e8-9c99-78a0c5b030cb.png">

<img width="1280" alt="screen shot 2018-07-11 at 12 26 32 pm" src="https://user-images.githubusercontent.com/24798364/42568652-c30f3240-8505-11e8-8520-ca880f979afd.png">

<img width="1280" alt="screen shot 2018-07-11 at 12 26 49 pm" src="https://user-images.githubusercontent.com/24798364/42568677-cd62d06c-8505-11e8-90a1-ce9e4f8ecaf2.png">

**Questions:**
N/A
